### DISCUSSION TITLE
help

### DISCUSSION THREAD

**Discussion by bronfenbrenner** [community/community#52484] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

hi. thanks for this amazing software for reddit.i have one problem. this software stuck at 25 post, plz tell me setting for a whole subreddit i mean all posts of a subreddit . thanks again

**Comment by airtower-luna** [comment]
It looks like you're looking for help with a software hosted on GitHub. In that case you should look for support information *in the repository for that software*, usually in the README or a file called SUPPORT. If those don't say anything, opening an issue there is usually good. :slightly_smiling_face:
